build-rs.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env sh
```

Comment:
I think it's worth restructuring the project in the future, once you will finish with your PRs. Each language has own specific build system, and it seems that we are going deeper and deeper into customization. So creating a folder for each language on the root level might be better than reusing java ecosystem convention of `src/main/x`. To be honest I didn't anticipate all the amazing contributions to this project so I couldn't foreseen that. :)